chain/cosmos/chain_node.go

```diff
@@ -521,17 +521,30 @@ func (tn *ChainNode) ExecTx(ctx context.Context, keyName string, command ...stri
 	if err != nil {
 		return "", err
 	}
+
 	output := CosmosTx{}
-	err = json.Unmarshal([]byte(stdout), &output)
+	err = json.Unmarshal(stdout, &output)
 	if err != nil {
 		return "", err
 	}
-	if output.Code != 0 {
-		return output.TxHash, fmt.Errorf("transaction failed with code %d: %s", output.Code, output.RawLog)
+
+	if err := testutil.WaitForBlocks(ctx, 1, tn); err != nil {
```

Comment:
this should also remain at 2. Moving to 1 causes issues in multi val environments due to docker + tendermint